keyboards/skyloong/gk68/pro/info.json

```diff
@@ -0,0 +1,380 @@
+            {"matrix": [1, 11], "x": 168, "y": 13, "flags": 4},
+            {"matrix": [1, 12], "x": 183, "y": 13, "flags": 4},
+            {"matrix": [1, 13], "x": 205, "y": 13, "flags": 4},
+            {"matrix": [1, 14], "x": 224, "y": 13, "flags": 4},
+
+            {"matrix": [2, 0], "x": 5, "y": 26, "flags": 4},
+            {"matrix": [2, 1], "x": 26, "y": 26, "flags": 4},
+            {"matrix": [2, 2], "x": 41, "y": 26, "flags": 4},
+            {"matrix": [2, 3], "x": 56, "y": 26, "flags": 4},
+            {"matrix": [2, 4], "x": 71, "y": 26, "flags": 4},
+            {"matrix": [2, 5], "x": 86, "y": 26, "flags": 4},
+            {"matrix": [2, 6], "x": 101, "y": 26, "flags": 4},
+            {"matrix": [2, 7], "x": 115, "y": 26, "flags": 4},
+            {"matrix": [2, 8], "x": 130, "y": 26, "flags": 4},
+            {"matrix": [2, 9], "x": 145, "y": 26, "flags": 4},
+            {"matrix": [2, 10], "x": 160, "y": 26, "flags": 4},
+            {"matrix": [2, 11], "x": 175, "y": 26, "flags": 4},
+            {"matrix": [2, 12], "x": 200, "y": 26, "flags": 4},
+            {"matrix": [2, 14], "x": 224, "y": 26, "flags": 4},
+
+            {"matrix": [3, 0], "x": 9, "y": 51, "flags": 4},
+            {"matrix": [3, 1], "x": 34, "y": 51, "flags": 4},
+            {"matrix": [3, 2], "x": 48, "y": 51, "flags": 4},
+            {"matrix": [3, 3], "x": 63, "y": 51, "flags": 4},
+            {"matrix": [3, 4], "x": 78, "y": 51, "flags": 4},
+            {"matrix": [3, 5], "x": 93, "y": 51, "flags": 4},
+            {"matrix": [3, 6], "x": 108, "y": 51, "flags": 4},
+            {"matrix": [3, 7], "x": 123, "y": 51, "flags": 4},
+            {"matrix": [3, 8], "x": 138, "y": 51, "flags": 4},
+            {"matrix": [3, 9], "x": 153, "y": 51, "flags": 4},
+            {"matrix": [3, 10], "x": 168, "y": 51, "flags": 4},
+            {"matrix": [3, 11], "x": 188, "y": 51, "flags": 4},
+            {"matrix": [3, 12], "x": 209, "y": 51, "flags": 4},
+            {"matrix": [3, 14], "x": 224, "y": 51, "flags": 4},
+
+            {"matrix": [4, 0], "x": 2, "y": 64, "flags": 4},
+            {"matrix": [4, 1], "x": 20, "y": 64, "flags": 4},
+            {"matrix": [4, 2], "x": 39, "y": 64, "flags": 4},
+            {"matrix": [4, 4], "x": 72, "y": 64, "flags": 4},
+            {"matrix": [4, 5], "x": 95, "y": 64, "flags": 4},
+            {"matrix": [4, 7], "x": 118, "y": 64, "flags": 4},
+            {"matrix": [4, 8], "x": 149, "y": 64, "flags": 4},
+            {"matrix": [4, 9], "x": 164, "y": 64, "flags": 4},
+            {"matrix": [4, 10], "x": 179, "y": 64, "flags": 4},
+            {"matrix": [4, 11], "x": 194, "y": 64, "flags": 4},
+            {"matrix": [4, 12], "x": 209, "y": 64, "flags": 4},
+            {"matrix": [4, 14], "x": 224, "y": 64, "flags": 4}
+        ],
+        "max_brightness": 200,
+        "hue_steps": 20,
+        "sat_steps": 20,
+        "speed_steps": 20,
+        "val_steps": 20
```

Comment:
```suggestion
        "val_steps": 20,
        "sleep": true
```

Reply:
I hope to continue to keep these Settings to inherit our company's setup habits.

Reply:
This change goes with the previous one removing `RGB_DISABLE_WHEN_USB_SUSPENDED` from config.h; they are just moving it to info.json.

Reply:
This doesnt remove anything?

`RGB_DISABLE_WHEN_USB_SUSPENDED` is now configured by `rgb_matrix.sleep` in `info.json`?